docs/python/editing.md

```diff
@@ -16,15 +16,24 @@ In this overview, we will describe the specific editing features provided by the
 
 ## Autocomplete and IntelliSense
 
-IntelliSense is a general term for code editing features that relate to code completion. Take a moment to look at the example below. When **print** is typed, notice how IntelliSense populates auto-completion options. The user is also given a list of options when they begin to type the variable named, **greeting**.
+IntelliSense is a general term for code editing features that relate to code completion. Take a moment to look at the example below. When **print** is typed, notice how IntelliSense populates auto-completion options. The user is also given a list of options when they begin to type the variable named **greeting**.
 
 ![Hello World Example for IntelliSense](images/editing/hello-world.gif)
 
 Autocomplete and IntelliSense are provided for all files within the current working folder. They're also available for Python packages that are installed in standard locations.
 
+### Auto Imports
+
+ Pylance offers auto import suggestions for modules on your workspace and/or packages you have installed on your environment. This enables import statements to be conviniently added automatically as you're typing.
```

Comment:
In instead of on? And a typo.
```suggestion
 Pylance offers auto import suggestions for modules in your workspace and/or packages you have installed in your environment. This enables import statements to be conveniently added automatically as you're typing.
```